### PR TITLE
Issue 45 new ping

### PR DESCRIPTION
**Pull request by eikenb**
Replace previous, broken, unmaintained ping library with a new working, maintained one.

See #45 for more.

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -363,10 +363,10 @@
...
 		},
 		{
-			"checksumSHA1": "DpsruMRlfaXitPiELMUatEOMh1k=",
-			"path": "github.com/tatsushid/go-fastping",
-			"revision": "d7bb493dee3e090e2ffb6914adddf17c1e7c026c",
-			"revisionTime": "2016-01-08T17:53:29Z"
+			"checksumSHA1": "oK0bJkEgq+zNzGEdU4j9AQcPbEs=",
+			"path": "github.com/sparrc/go-ping",
+			"revision": "4e5b6552494c8005c60de6c60b50ebaefc69e592",
+			"revisionTime": "2019-06-05T20:23:35Z"
```
Comment (banks) [inline vendor/vendor.json]: We should probably update this to use go mod soon but let's leave that for another PR!

Comment (eikenb) [reply vendor/vendor.json]: Yeah... tried that and to many things broke. I wanted to have everything working before doing it. So it's next after this.
